### PR TITLE
Add vscode configurations.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (kenzieschmoll) [inline packages/devtools_app/.vscode/launch.json:4]: is this comment supposed to be there?

File: packages/devtools_app/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
```
Comment (kenzieschmoll) [inline packages/devtools_app/.vscode/launch.json:7]: is there a profile mode configuration we can add?

File: packages/devtools_app/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        {
+            "name": "memory/default",
+            "cwd": "devtools_app",
+            "request": "launch",
+            "type": "dart",
+            "program": "test/test_infra/scenes/memory/default.stager_app.dart"
+        },
+        {
+            "name": "memory/diff_snapshot",
```
Comment (kenzieschmoll) [inline packages/devtools_app/.vscode/launch.json:22]: can you also add the performance screen stager as a configuration?

File: packages/devtools_app/.vscode/launch.json (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/.vscode/launch.json]: Yes, they contain usable information.

File: packages/devtools_app/.vscode/launch.json (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/.vscode/launch.json]: added

File: packages/devtools_app/.vscode/launch.json (hunk 3 above)
Comment (polina-c) [reply packages/devtools_app/.vscode/launch.json]: added

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/.vscode/launch.json (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/.vscode/launch.json]: okay, I'm surprised this isn't causing issues with json validation for this file

**Pull request merged**